.github/workflows/build.yml

```diff
@@ -31,6 +32,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Install ninja-build tool
+        if: runner.os != 'Linux'
```

Comment:
IMO `if: ${{ runner.os == 'Linux' }}` is better, the reason is that it forces you to use ${{ ... }} always, and why always use it? because `if: !negate.some.var` doesn't work as ! is a special charecter in YAML and there might be others.

Reply:
Since I'm doing the negation here, do you mean:
```
if: ${{ runner.os != 'Linux' }}
```
?

Reply:
AFAIK your code is fine, iirc I had trouble when the `! ` was negating a variable, and found out it was due YAML parser, so seemed wise to always use ${{ ... }} to avoid wasting time debugging something subtle to the YAML spec...

Reply:
Tests passed so I guess at the middle of the string it's fine, it's just that it's a pain to debug something that looks fine but it's not due the way YAML is parsed...

Reply:
Got it. In any case, I like the suggestion to be more defensive. I'll change it.